client/index.html

```diff
@@ -94,31 +94,30 @@ <h1 class="title">Connect</h1>
 									<h2>Network settings</h2>
 								</div>
 								<div class="col-sm-3">
-								<label>Name</label>
-									</div>
+									<label for="connect:name">Name</label>
+								</div>
 								<div class="col-sm-9">
-									<input class="input" name="name" value="<%= defaults.name %>">
+									<input class="input" id="connect:name" name="name" value="<%= defaults.name %>">
```

Comment:
I agree with @astorije there. I totally didn't think at all about ever wanting to use those names in CSS selectors. I guess the only real option here is to use `_` as everything else seems to be reserved (and `-` is used as a space replacement since we don't CamelCase CSS names). I wish there was a better looking character for representing namespaces that's not so close to `-` but apparently it's `[a-zA-Z0-9_-]`. Too bad, `:` made it easy to avoid id conflicts and looked good 😞 We could always escape it but it quickly gets ugly, especially in JS strings due to double escaping: `$("#connect\\:name")`.